src/components/AddressInput/AddressInput.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+const AddressInput = () => {
```

Comment:
I don't think this is the `AddressInput` component, I think this is the `AddressForm`.

On top of that, this uses an address field but it's for a location search, so how's about `LocationSearchForm`?